src/Rules/Program.hs

```diff
@@ -55,6 +55,9 @@ buildProgram rs = do
                         (True, s) | s > Stage0 -> do
                             srcDir <- buildRoot <&> (-/- (stageString Stage0 -/- "bin"))
                             copyFile (srcDir -/- takeFileName bin) bin
+                        (False, s) | s > Stage0 && takeFileName bin == "unlit" -> do
```

Comment:
Should we instead compare packages, i.e. `s > Stage0 && package == unlit`? This should be more robust.

Reply:
Done.